funnel/forms/project.py

```diff
@@ -89,6 +89,16 @@ def validate_bg_color(self, field):
         if not valid_color_re.match(field.data):
             raise forms.ValidationError("Please enter a valid color code")
 
+    # We run `strftime()` on `date` and `date_upto` for generating `datelocation`.
+    # And `strftime()` canot handle dates before year 1900.
```

Comment:
This was fixed in Python 3.2. Another reason to upgrade. https://bugs.python.org/issue1777412